Kino/AppDelegate.swift

```diff
@@ -13,27 +13,33 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        window = UIWindow(frame: UIScreen.main.bounds)
+        window?.rootViewController = setupMainNavigationViewController()
+        window?.makeKeyAndVisible()
+        
+        return true
+    }
+    
+    private func setupMainNavigationViewController() -> UINavigationController {
+        let movies = MainViewController(nibName: "MainViewController", bundle: nil)
+        movies.tabBarItem = UITabBarItem(title: nil, image: UIImage(systemName: "movieclapper"), tag: 0)
         
-        func createNavController(for rootViewController: UIViewController, title: String, imageName: String) -> UINavigationController {
-            rootViewController.tabBarItem = UITabBarItem(title: nil, image: UIImage(systemName: imageName), tag: 0)
-            rootViewController.navigationItem.title = title
-            rootViewController.navigationItem.leftBarButtonItem = UIBarButtonItem(image: UIImage(systemName: "list.star"), style: .plain, target: self, action: #selector(leftButtonTapped))
-            rootViewController.navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage(systemName: "bell.badge"), style: .plain, target: self, action: #selector(rightButtonTapped))
-            return UINavigationController(rootViewController: rootViewController)
-        }
+        let ticket = TicketsViewController(nibName: "TicketsViewController", bundle: nil)
+        ticket.tabBarItem = UITabBarItem(title: nil, image: UIImage(systemName: "ticket"), tag: 1)
         
-        let movies = createNavController(for: MainViewController(nibName: "MainViewController", bundle: nil), title: "KinkiNO", imageName: "movieclapper")
-        let tickets = createNavController(for: TicketsViewController(nibName: "TicketsViewController", bundle: nil), title: "Tickets", imageName: "ticket")
-        let favourites = createNavController(for: FavouritesViewController(nibName: "FavouritesViewController", bundle: nil), title: "Favourites", imageName: "heart.fill")
+        let fav = FavouritesViewController(nibName: "FavouritesViewController", bundle: nil)
+        fav.tabBarItem = UITabBarItem(title: nil, image: UIImage(systemName: "heart.fill"), tag: 2)
         
         let tabBarController = UITabBarController()
-        tabBarController.viewControllers = [movies, tickets, favourites]
+        tabBarController.viewControllers = [movies, ticket, fav]
         
-        window = UIWindow(frame: UIScreen.main.bounds)
-        window?.rootViewController = tabBarController
-        window?.makeKeyAndVisible()
+        let navigationController = UINavigationController(rootViewController: tabBarController)
         
-        return true
+        tabBarController.navigationItem.leftBarButtonItem = UIBarButtonItem(image: UIImage(systemName: "list.star"), style: .plain, target: self, action: #selector(leftButtonTapped))
```

Comment:
When the line is long, place each parameter on a separate line

Reply:
done